Specs/Scene/ImageryLayerCollectionSpec.js

```diff
@@ -433,6 +433,7 @@ defineSuite([
                     expect(features.length).toBe(1);
                     expect(features[0].name).toEqual('Foo');
                     expect(features[0].description).toContain('Foo!');
+                    expect(features[0].imageryLayer).toBeInstanceOf(ImageryLayer);
```

Comment:
It would be better to check for a particular `ImageryLayer` instance, rather than just checking it's the right type.  `addImageryprovider` creates and returns an `ImageryLayer`, so you just need to save that to a variable above and then check for it here.

Even better, modify the `returns features from two layers` test below to verify that the correct layers are associated with the correct features.